tests/commands/completion/fixtures/hello_command.py

```diff
@@ -1,13 +1,18 @@
 from __future__ import annotations
 
 from cleo.commands.command import Command
+from cleo.helpers import option
 
 
 class HelloCommand(Command):
-    """
-    Complete me please.
-
-    hello
-        { --dangerous-option= : This $hould be `escaped`. }
-        { --option-without-description }
-    """
+    name = "hello"
+    options = [
+        option(
+            "dangerous-option",
+            flag=False,
+            default=" ",
+            description="This $hould be `escaped`.",
+        ),
+        option("option-without-description", flag=False),
```

Comment:
```suggestion
        option("option-without-description"),
```

Reply:
My understanding is that `flag` is an option with one dash, like `-G`, and the "not flag" option is `--group`.

Reply:
Ah flag is just an option that doesn't accept a value: https://github.com/python-poetry/cleo/blob/efb274f445a939ee805f93ec7b8cd73cba5337e8/README.md?plain=1#L356

Reply:
That's something to remember when working on new docs. If we are confused, how about users that don't know the project :sweat_smile: